.github/container/Dockerfile.maxtext

```diff
@@ -86,5 +45,5 @@ RUN pip-finalize.sh
 
 WORKDIR ${SRC_PATH_MAXTEXT}
 
-# When tftext and lingvo wheels are published on pypi.org, revert this
+# When tftext wheels are published on pypi.org, revert this
 # Dockerfile to 5c4b687b918e6569bca43758c346ad8e67460154
```

Comment:
Remove both lines and do that revert?

Reply:
I cannot do that now: the commit mentioned here assumes that both `tftext` and `lingvo` are available on python-3.12, but it's not true for `lingvo`, that's why we still need to build `lingvo` for both x86_64 and arm64.